Os/test/ut/file/FileRules.cpp

```diff
@@ -256,6 +258,7 @@ bool Os::Test::File::Tester::OpenBaseRule::precondition(const Os::Test::File::Te
 
 void Os::Test::File::Tester::OpenBaseRule::action(Os::Test::File::Tester &state  //!< The test state
 ) {
+    printf("--> Rule: %s \n", this->getName());
```

Comment:
Should this be implemented at the STest level? It seems pretty convenient

How does one get access to those logs btw? I am assuming it gets captured by GoogleTest and put in a log file somewhere?